docker-compose.yml

```diff
@@ -1,32 +1,26 @@
 name: movies
 
-endpoints:
-  - service: frontend
-    path: /
-    port: 80
-  - service: api
-    path: /api
-    port: 8080
-
 services:
 
   frontend:
+    scale: 8
     build: frontend
     ports:
-      - 80
+      - 80:80
     volumes:
       - frontend:/src
     command: yarn start
 
   api:
+    scale: 4
```

Comment:
undo before final merge

Reply:
completely and remove the line or set to 1 or ...